### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by mkaraki**
This PR...

## Changes / fixes

- Add Japanese translation into i18n dir

## Screenshots (if applicable)

image

## Checklist

_Ensure you have checked off the following before submitting your PR._

- [x] tested locally with the [latest release of Hugo](https://github.com/gohugoio/hugo/releases). This requirement is [a standard](https://github.com/gohugoio/hugoThemes#theme-maintenance)
- [x] added new dependencies (Nothing to do)
- [ ] updated the [docs]() ⚠️ (Nothing to do?)

### CODE REVIEW

**Comment by rootwork** [comment]
Not knowing Japanese, I can only assume these translations are accurate.

@mkaraki Can you sign-off your code to [satisfy the DCO check](https://developercertificate.org/)? You can do this by appending `-s` to your `git commit`. [More details on this is in the check.](https://github.com/chipzoller/hugo-clarity/pull/355/checks?check_run_id=7519884015)

File: i18n/ja.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+[read_more]
+other = "続きを読む"
+
+[copyright]
+other = "Copyright"
```
Comment (rootwork) [inline i18n/ja.toml:5]: Translation for this?

File: i18n/ja.toml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+[all_rights]
+other = "All Rights Reserved"
```
Comment (rootwork) [inline i18n/ja.toml:8]: Translation for this?

**Review by rootwork — CHANGES_REQUESTED** [review]

File: i18n/ja.toml (hunk 2 above)
Comment (mkaraki) [reply i18n/ja.toml:8]: In Japan, `Copyright` and `All Rights Reserved` will not translated (same as Chinese). So, I kept this.

You can check this with Japanese Government's homepage like [this](https://www.kunaicho.go.jp/) or [this](https://www.mlit.go.jp/).

**Comment by mkaraki** [comment]
@rootwork 
In Japanese, almost all site use `Copyright (c) year name. All right reserved` in footer of page. So I didn't made translated it. (I think Chinese translation is same)

I did `-s --amend` and I think now it's pass DCO check.

**Comment by KaitaKoinuma** [comment]
I'm Japanese, these translations seem correct to me!
 I'm looking forward to merging this PR.

**Comment by rootwork** [comment]
@mkaraki  I'm not sure why the `git -s --amend` push didn't satisfy the DCO check -- maybe you can't amend a signoff? -- but in any case given that you were obviously trying to thus signoff the DCO I'm going to manually approve it and merge. Thanks for your contribution!

**Pull request merged**